src/accessibility.md.njk

```diff
@@ -6,31 +6,79 @@ layout: layout-single-page-prose.njk
 
 # Accessibility
 
-The GOV.UK Design System team works hard to ensure that this Design System and GOV.UK Frontend, the codebase it uses, are accessible.
+The GOV.UK Design System website and the codebase it uses, GOV.UK Frontend, is maintained by a team at the Government Digital Service (GDS).
+
+This page explains how the team works to ensure the Design System and Frontend are accessible.
+
+## Accessibility statement for the GOV.UK Design System website
```

Comment:
Can you lose the 'GOV.UK's from this point? (If I've understood the style correctly - it's ok to lose the GOV.UK after the first mention?)

Reply:
Yes, good point. I'll do that now.